core/src/main/java/com/datastrato/gravitino/Configs.java

```diff
@@ -11,11 +11,15 @@
 public interface Configs {
 
   String DEFAULT_ENTITY_STORE = "kv";
+  String RELATION_ENTITY_STORE = "rel";
```

Comment:
I would suggest to rename all "rel" and "relation" to "relational". According to Wikipedia (https://en.wikipedia.org/wiki/Relational_database), it is the relational database not the relation database.

Reply:
done